control-plane-operator/controllers/hostedcontrolplane/cno/clusternetworkoperator.go

```diff
@@ -542,12 +587,43 @@ if [[ -n $sc ]]; then kubectl --kubeconfig $kc delete --ignore-not-found validat
 				{Name: "konnectivity-proxy-ca", MountPath: "/etc/konnectivity/proxy-ca"},
 			},
 		},
+		{
+			// Token minter container
+			Command: []string{
+				"/usr/bin/control-plane-operator",
+				"token-minter",
+			},
+			Args: []string{
+				"--service-account-namespace",
+				workerNamespace,
+				"--service-account-name",
+				workerServiceAccount,
+				"--token-file",
+				fmt.Sprintf("/var/client-token/%s", tokenFile),
+				"--token-audience",
+				params.IssuerURL,
+				"--kubeconfig",
+				fmt.Sprintf("/etc/kubernetes/%s", kas.KubeconfigKey),
+			},
+			Name:                     tokenMinterContainerName,
+			Image:                    params.Images.TokenMinter,
+			ImagePullPolicy:          corev1.PullIfNotPresent,
+			Resources:                cnoResources,
```

Comment:
this should have its own resources and not reuse the cnoResources
Basically the goal is to preserve the resources that were set in the deployment before reconcile (if they've been dynamically changed)